README.md

```diff
@@ -563,8 +563,7 @@ end
 Then you can get this error message in exception handler:
 ```ruby
 rescue_from Pundit::NotAuthorizedError do |e|
-  message = e.reason ? I18n.t("pundit.errors.#{e.reason}") : e.message
-  flash[:error] = message, scope: "pundit", default: :default
+  flash[:error] = e.reason ? I18n.t("errors.#{e.reason}", scope: "pundit") : e.message
```

Comment:
since `"errors.#{nil}" == "errors."`, I think you can drop the ternary altogether 
```suggestion
  flash[:error] = I18n.t("errors.#{e.reason}", scope: "pundit", default: e.message)
```